lib/rules/no-dupe-class-members.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
What happens when there are duplicate class members? I assume this should be a "problem".

Reply:
Yeah, you're probably right. Let's see what others think.